charts/primary-site/templates/garbage-collector-cronjob.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: batch/v1
```

Comment:
Why are all these moved out of the folders? Is there some problem with the folders?